schemas/2.3.0.json

```diff
@@ -9,7 +9,7 @@
   ],
   "additionalProperties": false,
   "patternProperties": {
-    "^x-[\\w\\d\\.\\-\\_]+$": {
+    "^x-[\\w\\d\\.\\x2d_]+$": {
```

Comment:
Good catch! 

Hmmm 🤔 

I guess the problem is because it is inside a matching list `[...]` 🤔 

Otherwise I guess we needed to do: `     "^x\\x2d[\\w\\d\\.\\x2d_]+$": {`

Reply:
Yes, you guessed correctly. It is because that `-` is inside a list.